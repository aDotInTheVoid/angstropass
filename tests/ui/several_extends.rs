#[angstropass::langs]
mod langs {
    #[extends(foo)]
    #[extends(bar)]
    mod baz {}
}

fn main() {}
