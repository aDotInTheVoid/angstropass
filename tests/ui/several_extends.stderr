error: only one #[extends] attribute is allowed
 --> tests/ui/several_extends.rs:4:5
  |
4 |     #[extends(bar)]
  |     ^^^^^^^^^^^^^^^
